apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -37,41 +42,58 @@ import {
   Scalars,
   Pool,
   PublishingGroup,
+  Maybe,
 } from "@gc-digital-talent/graphql";
 
-import { categorizeSkill } from "~/utils/skillUtils";
 import {
   formatClassificationString,
   getClassificationGroup,
-  getFullPoolTitleLabel,
+  getClassificationSalaryRangeUrl,
+  getFullPoolTitleHtml,
+  getShortPoolTitleLabel,
   isAdvertisementVisible,
 } from "~/utils/poolUtils";
-import { wrapAbbr } from "~/utils/nameUtils";
 import SEO from "~/components/SEO/SEO";
 import Hero from "~/components/Hero/Hero";
 import useRoutes from "~/hooks/useRoutes";
 import useBreadcrumbs from "~/hooks/useBreadcrumbs";
 import EducationRequirements from "~/components/EducationRequirements/EducationRequirements";
 import useRequiredParams from "~/hooks/useRequiredParams";
 
-import ApplicationLink from "./components/ApplicationLink";
+import ApplicationLink, {
+  ApplicationLinkProps,
+} from "./components/ApplicationLink";
 import Text from "./components/Text";
 import SkillAccordion from "./components/SkillAccordion";
 import DataRow from "./components/DataRow";
 import GenericJobTitleAccordion from "./components/GenericJobTitleAccordion";
 import DeadlineDialog from "./components/DeadlineDialog";
+import WorkLocationDialog from "./components/WorkLocationDialog";
```

Comment:
Out of scope for your PR but the deadline dialog is triggering a dom warning.  If you're up for it, it could be a quick fix.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/9aa63c0a-7a0e-4098-ad22-af15c905e836)

Reply:
Oh yeah, you totally cannot do that 🤦‍♀️ Fixed in [fix invalid HTML](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9145/commits/5e622bcf962bcc293a51ae9b99e826c87a9f42b6)

Reply:
Weird.  I see the fix but I'm still getting the warning.  I scanned the file and I'm not sure where it's coming from.  :confused:

Reply:
I'll try and dig deeper to see if I can figure it out

Reply:
Looks like it was the `DeadlineDialog` Fixed in [fix invalid dom nesting](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9145/commits/9b7a9b126f6540d91eee62340fc1328d121dfebc)